src/game.zig

```diff
@@ -326,6 +326,20 @@ pub const collision = struct {
 
 pub const Gamemode = enum(u8) { survival = 0, creative = 1 };
 
+pub const DamageType = enum(u8) {
+	none = 0, // For when you are adding health
```

Comment:
Should be called healing then. And yes dying from healing might make sense with some status effects.